modules/statics/src/coins.ts

```diff
@@ -1328,6 +1328,14 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.HNT,
     SOL_TOKEN_FEATURES
   ),
+  tofcsolToken(
+    '20b20bc7-86b8-4f58-a8e9-a7cedbc2a507',
+    'ofctsol:gari',
+    'testnet GARI',
```

Comment:
The formatting seems off here?

```suggestion
    'testnet Gari Token',
```

Reply:
updated